.pre-commit-config.yaml

```diff
@@ -37,7 +37,9 @@ repos:
       - id: check-rebase
         args:
           - git://github.com/packit-service/ogr.git
+        stages: [manual, push]
   - repo: https://github.com/packit/requre
     rev: 0.2.2
     hooks:
       - id: requre-purge
+        stages: [manual]
```

Comment:
@lachmanfrantisek `push` too at least? if we can't use `commit` because of the CI?

Reply:
Yes, that would mean we run it on push and in zuul.